experimental/table/client/src/main/java/alluxio/client/catalog/RetryHandlingCatalogMasterClient.java

```diff
@@ -105,7 +105,7 @@ public TableInfo getTable(String databaseName, String tableName) throws AlluxioS
   public boolean attachDatabase(String dbName, Map<String, String> configuration)
       throws AlluxioStatusException {
     return retryRPC(() -> mClient.attachDatabase(
-        AttachDatabasePRequest.newBuilder().build().newBuilder().setDbName(dbName)
+        AttachDatabasePRequest.newBuilder().setDbName(dbName)
```

Comment:
you will have to rebase, to get the updated line.

Reply:
This was on the latest master though? Did you fix it in another branch?

Reply:
whoops nevermind. I guess you changed it around when I submitted the PR